winbuild/Makefile.vc

```diff
@@ -21,10 +22,20 @@ CFGSET=true
 !MESSAGE                                  Libraries can be fetched at http://pecl2.php.net/downloads/php-windows-builds/
 !MESSAGE                                  Uncompress them into the deps folder.
 !MESSAGE   WITH_SSL=<dll or static>     - Enable OpenSSL support, DLL or static
+!MESSAGE      SSL_LIBS=<File1 File2>    - OpenSSL libs to use (default: libeay32.lib ssleay32.lib) (requires WITH_SSL)
+!MESSAGE      SSL_HEADER_PATH=<path>    - Path to OpenSSL include files. (requires WITH_SSL)
+!MESSAGE      SSL_CFLAGS=...            - Extra flags to supply when compiling OpenSSL modules (requires WITH_SSL)
```

Comment:
....and I'm not sure what you mean by the second comment

Taking a step back I need to be able to specify 

- The  lib **NAME** for the downstream dependencies (zlib/OpenSSL)
- The lib **PATH** for same
- The header **PATH* for same

I currently do that by overloading the internal variables XYZ_LDFLAGS and XYZ_CFLAGS, but if it makes is more useful for others to supply different parameterization (XYZ_LIB, XYZ_LDPATH, XYH_HEADERPATH?) I can do that.

I'm just seeking to have something I can use without having to pull the  `MAKE="NMAKE /e"` trick.  otoh it has to be generally useful